packaging/debian/control

```diff
@@ -37,7 +37,7 @@ Build-Depends: debhelper (>= 10~),
                libsnappy-dev,
                libsnmp-dev [linux-any hurd-i386],
                librdkafka-dev (>= 1.1.0) <!sng-nokafka>,
-               openjdk-8-jdk <!nojava>,
+               openjdk-11-jdk <!nojava>,
```

Comment:
Could we use `default-jdk-headless` instead ? The goal is not to specify java version, and headless just why not.

Reply:
The lighter, the better. :)